edn/src/types.rs

```diff
@@ -214,12 +264,6 @@ impl Value {
     }
 }
 
-impl From<f64> for Value {
```

Comment:
Oh, I see, you made these macros.  For just a few implementations I think this trickery is overkill, but it's your call.